.github/workflows/create-tag.yaml

```
@@ -14,7 +14,7 @@ defaults:
 jobs:
   Create-Tag:
     runs-on: ubuntu-18.04
-    if: github.repository == 'rook/rook'
+    if: github.repository == 'rook/rook' && contains(secrets.ADMIN, github.actor)
```

```suggestion
    if: github.repository == 'rook/rook' && contains(secrets.ADMIN, github.actor)
```
@travisn you need to add the secrets name `ADMIN` similar to `GITHUB_TOKEN` which will contain the GitHub username of the users who are allowed to trigger this action.
fyi how contains works(https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#contains)


Also, I have not tested so once this is merged we can test accordingly.

---

How about if we make a secret very specific to releases. I've created this secret now.
```suggestion
    if: github.repository == 'rook/rook' && contains(secrets.RELEASE_ADMINS, github.actor)
```

---

done